db/db.go

```diff
@@ -0,0 +1,34 @@
+package db
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+func SetupDB() {
+	var err error
+
+  dsn := getDSN()
+  DB, err = gorm.Open(mysql.Open(dsn), &gorm.Config{})
+
+	if err != nil {
```

Comment:
missed identy here

Reply:
Looks like vscode (or me hahah) used spaces instead of indentations :thinking: 
Will fix it, thanks!

Reply:
hahahhaa no problem. Can help you https://code.visualstudio.com/docs/languages/go.